ckanext/csrf_filter/anti_csrf.py

```diff
@@ -179,7 +179,6 @@ def get_cookie_token():
     """
     token = request_helpers.get_cookie(TOKEN_FIELD_NAME)
     if is_valid_token(token):
-        LOG.debug("Obtaining token from cookie")
```

Comment:
could you set them to trace or they are not needed at all?

Reply:
Trace-level logging doesn't appear to be a thing with this library.